### PR TITLE
#225 "Search Products" - sort results by ProductID and Status (5)

### PR DESCRIPTION
**Pull request by alexejFroebel**
### Definition of done:
- [x] The feature works -> production build
- [x] Tests exist and run successfully
- [x] All Tests work
- [x] Happy Path covered
- [x] No new Bugs/Followups

- [x] Documentation:
  - Readme.md
  - Doc-Section in PIT
  - Groovy/OpenAPI

---

### Release Types:
minor
### Release notes:
- The following endpoints are now sortable:
 - attribute
 - attributeSection
 - contract
 - manufacturer
 - priceType
 - supplier
 - unitOfMeasure
 - classificationGroup/product
 - classificationGroup
 - catalog
 - catalogProduct
 - classification 
---
## Code Review Checklist:
- [x] Does the code work?
- [x] Is all the code easily understood?
- [x] Are the function signatures named properly (function name, parameter names?)
- [x] Is the spelling correct?

- [x]  Is there redundant or duplicate code?
- [x]  Is there any commented out code?
- [x]  Can any logging or debugging code be removed?
- [x]  Is `for(i)` or `while` used? If so, remove
- [x]  Can any of the code be replaced with library functions?

-  Do you check input parameters (such as route parameters) for:
- [x] Type
- [x] Length
- [x] Format
- [x] Range?

- [x] Can comments be removed?
- [x] Is "external functionality" (API, Readme.md etc) documented/updated?
- [x] Is any unusual behavior or edge case handling described?

- [x] Do tests exist?
- [x] Do the snapshots look correct?
- [x] Are tests comprehensive?
- [x] Do unit tests actually test that the code is performing the intended functionality?

### CODE REVIEW

File: customizationArea/server/serviceMethods/pimApi.groovy [hunk 1]
```diff
@@ -162,6 +162,11 @@ class PitGroovyApi {
 
     /**
      * Retrieve a classification
+     * @param classificationId ClassificationId
+     * @param options Optional
+     * <p>
+     * include: fields which should be included
```
Comment (UweStolz) [inline customizationArea/server/serviceMethods/pimApi.groovy]: Inconsistent to the other parameter

File: customizationArea/server/serviceMethods/pimApi.groovy [hunk 2]
```diff
@@ -182,6 +187,15 @@ class PitGroovyApi {
 
     /**
      * Retrieve a classificationGroup
+     * @param  classificationId ClassificationId
+     * @param  classificationGroupId ClassificationGroupId
+     * @param  options Optional
+     * <p>
+     * Key: sort - the field which should be used for sorting. Available values are classificationGroupId and statusId
```
Comment (UweStolz) [inline customizationArea/server/serviceMethods/pimApi.groovy]: Capitalize, please
